packages/manager/modules/pci/src/projects/project/storages/cold-archive/cold-archives.constants.js

```diff
@@ -1,30 +1,31 @@
 export const COLD_ARCHIVE_TRACKING_PREFIX =
   'PublicCloud::pci::projects::project::storages::cold_archive';
 
-export const REGION = 'RBX-ARCHIVE';
+export const REGION = 'rbx-archive';
 
 export const DATE_FORMAT = 'DD/MM/YYYY HH:mm';
 
 export const CHECK_PRICES_DOC_LINK = {
-  DEFAULT: 'https://www.ovhcloud.com/en/public-cloud/prices/#',
-  DE: 'https://www.ovhcloud.com/de/public-cloud/prices/#',
-  AU: 'https://www.ovhcloud.com/en-au/public-cloud/prices/#',
-  CA: 'https://www.ovhcloud.com/en-ca/public-cloud/prices/#',
-  GB: 'https://www.ovhcloud.com/en-gb/public-cloud/prices/#',
-  IE: 'https://www.ovhcloud.com/en-ie/public-cloud/prices/#',
-  SG: 'https://www.ovhcloud.com/en-sq/public-cloud/prices/#',
-  ES: 'https://www.ovhcloud.com/es-es/public-cloud/prices/#',
-  QC: 'https://www.ovhcloud.com/fr-ca/public-cloud/prices/#',
-  FR: 'https://www.ovhcloud.com/fr/public-cloud/prices/#',
-  MA: 'https://www.ovhcloud.com/fr-ma/public-cloud/prices/#',
-  SN: 'https://www.ovhcloud.com/fr-sn/public-cloud/prices/#',
-  TN: 'https://www.ovhcloud.com/fr-tn/public-cloud/prices/#',
-  IT: 'https://www.ovhcloud.com/it/public-cloud/prices/#',
-  NL: 'https://www.ovhcloud.com/nl/public-cloud/prices/#',
-  PL: 'https://www.ovhcloud.com/pl/public-cloud/prices/#',
-  PT: 'https://www.ovhcloud.com/pt/public-cloud/prices/#',
-  WS: 'https://www.ovhcloud.com/es/public-cloud/prices/#',
-  WE: 'https://www.ovhcloud.com/en/public-cloud/prices/#',
+  DEFAULT: 'https://www.ovhcloud.com/en/public-cloud/prices/#11500',
```

Comment:
**question:** `#11500` is it valid?

Reply:
Yes, it is the anchor to point to the cold archive section in the pricing page.
The cold archive section is not in prod yet so it's normal that the anchor doesn't work yet.